tests/browser/wrap-history.browser.js

```diff
@@ -23,25 +23,6 @@ jil.browserTest('history functions are wrapped', function (t) {
   t.end()
 })
 
-jil.browserTest('a new property is not shown in the history object', function (t) {
```

Comment:
Instead of removing this test, can you just modify it to match the behavior you are expecting to see so that we can continue to validate this case for the future

Reply:
Done